### ISSUE TITLE
test: HTTP `POST` action in parallel_workload

### ISSUE THREAD

**Issue by sploiselle** [MaterializeInc/materialize#20673]
### Feature request

In #20472, we discovered it would have been nice to have an action that we can send not as a SQL command, but as an HTTP `POST`. It looks like `parallel_workload` is set up to exclusively support SQL commands, though, so we had to bail on that approach.

HTTP `POST` would be useful for both webhook sources and the HTTP APIs. I imagine in the future we could also support `DELETE`, `PUT`, etc.

Pinging @philip-stoev just for prioritization.

**Comment by def-** [comment]
Yes, definitely makes sense to extend parallel-workload with non-SQL features we support like this. I'll put this on my plate later.
